### PR TITLE
Update IPublishedContentQuery to use proper standards

### PR DESCRIPTION
**Pull request by Zeegaan**
# Notes
- Changed code examples from being directly in view to being in a service

### CODE REVIEW

**Comment by Zeegaan** [comment]
This will again need a internal review from the D-team 👍

**Comment by sofietoft** [comment]
I have a quick question on this one, based on a conversation with the community docs team earlier today.

The first code sample you've updated - that still goes into a template/view, right?
Just wondering, as the initial intro to the article mentions the templates - also, where we link to this article also mentions strongly types and templates. If we're changing this, we need to change it the right way 💪

**Comment by Zeegaan** [comment]
Hey @sofietoft, no this is all supposed to be backend code, but I do see we still mention templates etc. 🤔 
I will have a look at cleaning this up so its clear 👍

**Comment by sofietoft** [comment]
Thanks for clearing that up @Zeegaan ! (and Happy Birthday btw 🥳 )

Does the service still need to be injected in a template/view somewhere?
If it does - should we add a sample for that as well?

Also, perhaps this is a bigger discussion around how we handle these best practice code samples in our docs in general 😄

**Comment by Zeegaan** [comment]
@sofietoft No, I have refactored our Quick start: https://our.umbraco.com/documentation/Reference/Searching/Examine/quick-start/ 
This now uses a custom view model to use in the view instead, this article then builds on how you'd use the IPublishedContentQuery in the SearhService. 
Now that i say that, I see that is not mentioned at all, so I will add that in aswell 🙈

**Comment by sofietoft** [comment]
Ah! Missed that one.

Cool, thanks! 😄

File: Reference/Querying/IPublishedContentQuery/index.md [hunk 1]
```diff
@@ -9,13 +9,25 @@ meta.Description: "Querying in views with IPublishedContentQuery in Umbraco"
 
 The `IPublishedContentQuery` interface contains various query methods for accessing strongly typed content in templates.
```
Comment (kjac) [inline Reference/Querying/IPublishedContentQuery/index.md:10]:
```suggestion
The `IPublishedContentQuery` interface contains various query methods for accessing strongly typed content in services etc.
```

**Comment by sofietoft** [comment]
Looks good 💪 

I just updated the text on the page linking to the "IPublishedContentQuery" article, using the same text as in the intro of the article.

**Pull request merged**